pkg/apis/configuration/v1alpha1/types.go

```diff
@@ -50,13 +50,20 @@ type GlobalConfigurationList struct {
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 // +kubebuilder:validation:Optional
 // +kubebuilder:resource:shortName=ts
+// +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="State",type=string,JSONPath=`.status.state`,description="Current state of the VirtualServer. If the resource has a valid status, it means it has been validated and accepted by the Ingress Controller."
+// +kubebuilder:printcolumn:name="Reason",type=string,JSONPath=`.status.reason`
+// +kubebuilder:printcolumn:name="IP",type=string,JSONPath=`.status.externalEndpoints[*].ip`
```

Comment:
shall we remove the IP and port print columns for now as well?

(crds will need to be re-generated after that)

Reply:
done